WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/AnnualSiteStatsUseCaseTest.kt

```diff
@@ -96,7 +93,7 @@ class AnnualSiteStatsUseCaseTest : BaseUnitTest() {
         selectedDate.set(Calendar.YEAR, 2019)
         selectedDate.set(Calendar.MONTH, Calendar.DECEMBER)
         selectedDate.set(Calendar.DAY_OF_MONTH, 31)
-        verify(selectedDateProvider, times(2)).selectDate(selectedDate.time, listOf(selectedDate.time), ANNUAL_STATS)
+        verify(selectedDateProvider, times(1)).selectDate(selectedDate.time, listOf(selectedDate.time), ANNUAL_STATS)
```

Comment:
I wanted to confirm if this is the expected change (from `times(2)` to `times(1)`)

Reply:
[SAME](https://github.com/wordpress-mobile/WordPress-Android/pull/17673#discussion_r1053130469)